dev/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM sgrio/java:jre_8_ubuntu 
+WORKDIR /home
+RUN apt-get update
+RUN apt-get install -y net-tools
+RUN apt-get install -y telnet
+RUN apt-get install -y wget xvfb unzip libgtk-3-dev
+# Set locale
+RUN apt-get clean && apt-get update && apt-get install -y locales
+RUN locale-gen en_US.UTF-8
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US:en
+ENV LC_ALL en_US.UTF-8
+RUN wget http://eclipse.mirror.rafal.ca/technology/epp/downloads/release/2019-09/R/eclipse-jee-2019-09-R-linux-gtk-x86_64.tar.gz
```

Comment:
Try the non-mirror:
http://www.eclipse.org/external/technology/epp/downloads/release/2019-09/R/eclipse-jee-2019-09-R-linux-gtk-x86_64.tar.gz
Or better is the bluemix mirror:
http://eclipse.bluemix.net/packages/2019-09/data/eclipse-jee-2019-09-R-linux-gtk-x86_64.tar.gz